financial-connections/src/main/java/com/stripe/android/financialconnections/features/partnerauth/PartnerAuthScreen.kt

```diff
@@ -5,10 +5,13 @@ import androidx.compose.runtime.State
 import com.airbnb.mvrx.compose.collectAsState
 import com.airbnb.mvrx.compose.mavericksViewModel
 import com.stripe.android.financialconnections.features.common.SharedPartnerAuth
+import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest.Pane
 
 @Composable
 internal fun PartnerAuthScreen(inModal: Boolean) {
-    val viewModel: PartnerAuthViewModel = mavericksViewModel()
+    val viewModel: PartnerAuthViewModel = mavericksViewModel(
+        argsFactory = { PartnerAuthViewModel.Args(inModal, Pane.PARTNER_AUTH) }
```

Comment:
is the idea to pass the pane type as args to viewmodel on all panes? what would be the benefit of it?

Reply:
Nope, just for screens that use `SharedPartnerAuthState`. We previously constructed the initial state in the factory’s `initialState()` method, but can’t do that now because we need to know the `isModal` value for the partner auth screens.